lib/repl.js

```diff
@@ -778,6 +780,22 @@ function filteredOwnPropertyNames(obj) {
   return Object.getOwnPropertyNames(obj).filter(intFilter);
 }
 
+function getGlobalLexicalScopeNames() {
+  if (inspector === false) return [];
+  try {
+    const session = new inspector.Session();
+    session.connect();
+    let names;
```

Comment:
I'd initialize `names` to an empty array so that if the inspector callback is actually called asynchronously things won't crash and burn.